src/views/partials/error_summary.njk

```diff
@@ -10,6 +10,12 @@
                         <li>
                             {{ errors.serverError.summary }}
                         </li>
+                    {% elseif errors[0].msg is not undefined %}
+                        {% for error in errors %}
+                            <li>
+                                <a href="#{{ error.fieldId }}">{{ error.msg }}</a>
+                            </li>
+                        {% endfor %}
```

Comment:
This is duplicating the functionality already implemented for error summaries.
The errors object should be an object with keys that are the field names and the value be an object with a summary and inine fields. 
See the error manifest in `src/utils/error_manifests/default.ts` for details.